README.md

```diff
@@ -1,14 +1,19 @@
 # fast-api-dbs
 
-## How to run?
+## Install using pip
 
 ```
-pip install -r requirements.txt
+pip install -e .
 ```
 
-```mermaid
-flowchart LR
-    a --> b[ucl] <--> c{test};
+## Install using poetry
+
+```
+poetry install
 ```
 
-## Git commit hook
+## How to run?
+
+```
+uvicorn app.main:app --reload
```

Comment:
There is another way to run in poetry with `poetry run uvicorn`. Please mention that.